### PR TITLE
Fixed some things and added a function

### PR DESCRIPTION
**Pull request by Artitus**
- Added start script to package.json so that the README isn't a lie anymore.
- Reformatted the command handler to have two commands instead of just one
- Added a clap command

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -37,6 +47,28 @@ client.on("message", (message) => {
 
         message.channel.send(lastMessage.author + ' ' + newMessageParts.join(''));
     });
-});
+}
+
+function clap(message, args) {
+    message.channel.fetchMessages()
+    .then(messages => {
```
Comment (tobbbles) [inline bot.js:54]:
```suggestion
        .then(messages => {
```
indent this 1 tab

File: bot.js [hunk 2]
```diff
@@ -37,6 +47,28 @@ client.on("message", (message) => {
...
+        const newMessageParts = [];
+        for (let i = 0; i < lastMessage.content.length; i++) {
+            if (lastMessage.content[i] == ' ') {
+                newMessageParts.push('👏');
+                continue;
+            }
+            
+                newMessageParts.push(lastMessage.content[i].toUpperCase());
+            
+            }
```
Comment (tobbbles) [inline bot.js:68]:
```suggestion
            }
```

File: bot.js [hunk 3]
```diff
@@ -37,6 +47,28 @@ client.on("message", (message) => {
...
+
+        const lastMessage = filteredMessages.first();
+        const newMessageParts = [];
+        for (let i = 0; i < lastMessage.content.length; i++) {
+            if (lastMessage.content[i] == ' ') {
+                newMessageParts.push('👏');
+                continue;
+            }
+            
+                newMessageParts.push(lastMessage.content[i].toUpperCase());
```
Comment (tobbbles) [inline bot.js:66]:
```suggestion
            
            newMessageParts.push(lastMessage.content[i].toUpperCase());
```

**Review by tobbbles — CHANGES_REQUESTED** [review]

**Comment by DLMousey** [comment]
LGTM - there's an existing issue with an exception being thrown if there's no target user provided but that's an existing issue so i'll fix that.

**Comment by Artitus** [comment]
Noice

**Pull request merged**